client/packages/invoices/src/Returns/modals/NewReturnItems/ReturnQuantitiesTable.tsx

```diff
@@ -0,0 +1,54 @@
+import {
+  CellProps,
+  DataTable,
+  NumberInputCell,
+  SupplierReturnLine,
+  useColumns,
+} from 'packages/common/src';
+import React from 'react';
+
+const QuantityToReturnCell = (props: CellProps<SupplierReturnLine>) => (
```

Comment:
Just wondering -- why create a new component that just wraps `NumberInputCell`?

If it's just so you can include it in the column definition below, then you can just inline in the definition:
```tsx
[
      'numberOfPacksToReturn',
      { Cell: (props) =>   <NumberInputCell {...props} isRequired max={props.rowData.availableNumberOfPacks}min={1}/>, width: 100, setter: updateLine },
],
```

Maybe that's messier? I dunno.

Reply:
Haha no I think you're right - originally had some logic in there and felt I should pull out, but now it doesn't add much value. Fixed: [68d5070](https://github.com/msupply-foundation/open-msupply/pull/2928/commits/68d507001841556a7f95870eaf9e862e130a2447)